ucrxe_dds_client: Implement simple parameter-driven message namespace (#25444)
PX4/PX4-Autopilot: docs/en/middleware/uxrce_dds.md

  - [UXRCE_DDS_SYNCT](../advanced_config/parameter_reference.md#UXRCE_DDS_SYNCT): Bridge time synchronization enable.
    The uXRCE-DDS client module can synchronize the timestamp of the messages exchanged over the bridge.
    This is the default configuration. In certain situations, for example during [simulations](../ros2/user_guide.md#ros-gazebo-and-px4-time-synchronization), this feature may be disabled.
  - [`UXRCE_DDS_NS_IDX`](../advanced_config/parameter_reference.md#UXRCE_DDS_NS_IDX): Index-based namespace definition

>     Setting this parameter to any value other than -1 defines a simple namespace based on a prefix (`uav_`) and an numeric index, e.g. `uav_0`, `uav_1`, etc.

Reviewer: I think you mean

```suggestion
    Setting this parameter to any value other than `-1` creates a namespace with the prefix `uav_` and the specified value  e.g. `uav_0`, `uav_1`, etc.
```

:::

- A simple index-based namespace can be applied by setting the parameter [`UXRCE_DDS_NS_IDX`](../advanced_config/parameter_reference.md#UXRCE_DDS_NS_IDX) to a value between 0 and 9999.
  This will generate a namespace such as `/uav_0`, `/uav_1`, and so on.

>   This technique is ideal if vehicles must be persistently associated with namespaces as their clients are automatically started through PX4.

Reviewer: ```suggestion
  This technique is ideal if vehicles must be persistently associated with namespaces because their clients are automatically started through PX4.
```

Author: Reworded as suggested

::: info
PX4 parameters cannot carry rich text strings.
Therefore, you cannot use [`UXRCE_DDS_NS_IDX`](../advanced_config/parameter_reference.md#UXRCE_DDS_NS_IDX) to automatically start a client with an arbitrary message namespace through PX4.

> Instead, you may put the appropriate command, e.g.

Reviewer: ```suggestion
You can however specify a namespace when starting the client, using the `-n` argument:
```
